pydarn/plotting/fan.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett, Marina Schmidt
+"""
+Fan plots, mapped to AACGM coordinates
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import aacgmv2
+import datetime as dt
+import os
+
+from pydarn import PyDARNColormaps
+from typing import List
+from matplotlib import ticker, cm, colors
+
+class fan():
+    # TODO: Add class documentation
+
+    def __str__(self):
+        return "This class is static class that provides"\
+                " the following methods: \n"\
+                "   - plot_fan()\n"\
+                "   - return_beam_pos()\n"
+
+    @classmethod
+    def plot_radar_fov(cls, dmap_data: List[dict], scan_index: int = 0,
+                       all_gates: bool = False, parameter: str = 'v',
+                       hemisphere: str = 'north', lowlat: int = 50,
+                       cmap: str = None, groundscatter: bool = False,
+                       zmin: int = None, zmax: int = None,
+                       colorbar: plt.colorbar = None,
+                       colorbar_label: str = ''):
+        # TODO: pyDARN can work out the hemisphere via superDARNRadars.radars
+        # check out the file. Even has an enum for hemisphere and associated
+        # value you will find in MAP files
+        """
+        Plots a radar FOV fan plot for the given data and scan number
+
+        Parameters
+        -----------
+            dmap_data: List[dict]
+                Named list of dictionaries obtained from SDarn_read
+            scan_index: int
+                Scan number from beginning of first record in file
+                Default: 0
+            parameter: str
+                Key name indicating which parameter to plot.
+                Default: v (Velocity). Alternatives: 'p_l', 'w_l', 'elv'
+            hemisphere: str
+                North or south, depending on radar. Currently,
+                pyDARN cannot work this out
```

Comment:
What do you mean pyDARN cannot work this out? Its in the SuperDARNRadars list.

Reply:
Resolved